docs/examples/use_cases/pytorch/efficientnet/image_classification/dataloaders.py

```diff
@@ -274,6 +274,190 @@ def expand(num_classes, dtype, tensor):
 
 
 class PrefetchedWrapper(object):
+    def prefetched_loader(loader, num_classes, one_hot):
```

Comment:
```suggestion
    @classmethod
    def prefetched_loader(loader, num_classes, one_hot):
```

Reply:
Fixed

Reply:
I'm terribly sorry, it should be `@staticmethod`.....